docs/user-guide.md

```diff
@@ -1,12 +1,50 @@
-<br>
+<br />
 
-### What does the app do?
-- 1Hive's Redemptions app allows Aragon organizations to grant their token holders the right to redeem tokens in exchange for a proportional share of the organizations treasury assets.
+## What is the Redemptions app?
 
-<br>
+Redemptions is an Aragon app that enables redeeming organizations tokens for organization assets.
 
-### How do you use it?
-- user guide TBD
-- ideally this will have a walkthrough of the apps features with screenshots showing how to interact with the app to do stuff
+The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn by contributing to the hive for the underlying asset of the vault.
```

Comment:
The app doesn't create a vault. It uses the already created vault of the dao.

Reply:
Good catch. Updated accordingly